gramps/gen/lib/locationbase.py

```diff
@@ -189,3 +189,44 @@ def set_county(self, data):
     def get_county(self):
         """Return the county name of the LocationBase object."""
         return self.county
+
+    def get_object_state(self):
```

Comment:
Why is it necessary to define the `get_object_sate` and `set_object_state` methods here?  They should already be inherited by the Primary or Secondary objects.